armi/bookkeeping/visualization/entryPoint.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+import sys
+import pathlib
+
+from armi import runLog
+from armi.cli import entryPoint
+
+
+class VisFileEntryPoint(entryPoint.EntryPoint):
+    """
+    Create visualization files from database files.
+    """
+
+    name = "vis-file"
+    description = "Convert ARMI databases in to visualization files"
+
+    _FORMAT_VTK = "vtk"
+    _SUPPORTED_FORMATS = {_FORMAT_VTK}
+
+    def __init__(self):
+        entryPoint.EntryPoint.__init__(self)
+
+        self.nodes = None
+
+    def addOptions(self):
+        self.parser.add_argument("h5db", help="Input database path", type=str)
+        self.parser.add_argument(
+            "--output-name",
+            "-o",
+            help="Base to use output file name(s)",
```

Comment:
help message grammar issue?

`Base name for output files. Will have suffixes appended for timesteps`

maybe?